src/frontend/js/widgets/Dashboard/components/DashboardItem/index.tsx

```diff
@@ -9,6 +21,7 @@ export type DashboardItemProps = PropsWithTestId<{
   imageFile?: Nullable<JoanieFile>;
   footer?: ReactNode;
   mode?: 'full' | 'compact';
+  more?: ReactNode;
```

Comment:
I would have create a dedicated component for this.
```suggestion
moreMenu?: MoreMenu
```

And it's little item children.
```ts
interface MoreMenuItemProps {
  label: string
  to?: string
  href?: string
}
const MoreMenuItem = ({label, href, to}) => (
  <li>
    <div className="selector__list__link">
      {/* wrap label in <a href>, <Link to> or just label */}
      {label}
    </div>
  </li>
)
```